src/featurestorebundle/orchestration/PostActionsRunner.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+from logging import Logger
+from box import Box
+from featurestorebundle.orchestration.PostActionInterface import PostActionInterface
+from daipecore.decorator.ContainerManager import ContainerManager
+
+
+class PostActionsRunner:
+    def __init__(self, logger: Logger, post_actions: Box):
+        self.__logger = logger
+        self.__post_actions = post_actions
+
+    def run(self):
+        post_action_services = self.__get_post_action_services()
+
+        self.__validate_post_action_services(post_action_services)
+
+        if len(self.__post_actions) == 0:
+            self.__logger.info("No post actions")
+
+        for post_action_service, post_action in zip(post_action_services, self.__post_actions):
```

Comment:
what's the difference between `post_action_services` and `self.__post_actions`?

Reply:
renamed to post_action_service_identifier